src/main/telemetry/telemetry.c

```diff
@@ -137,6 +137,9 @@ void telemetryCheckState(void)
 #ifdef TELEMETRY_SRXL
     checkSrxlTelemetryState();
 #endif
+#ifdef TELEMETRY_IBUS
+    checkIbusTelemetryState();
```

Comment:
yes, makes total sense..
What do we do in this case? add another condition based on the flash size in the common target file?